package.json

```diff
@@ -16,7 +16,7 @@
     "lint:fix": "vue-cli-service lint",
     "start": "node app.js",
     "start:client": "concurrently \"ENABLE_VUE_DEVTOOLS=true vue-cli-service serve\" \"npm run start:devtools\"",
-    "start:dev": "concurrently \"npm run start:server\" \"ENABLE_VUE_DEVTOOLS=true npm run start:client\" \"npm run start:devtools\"",
```

Comment:
Small issue with the way we were loading the client. With our previous change to fix the client, starting dev now called vue dev tools twice.